packages/trie/src/baseTrie.ts

```diff
@@ -270,9 +288,10 @@ export class Trie {
     let value = null
     let foundNode = null
     value = await this.db.get(node as Buffer)
-
     if (value) {
       foundNode = decodeNode(value)
+    } else {
+      throw new Error('Missing node in DB')
```

Comment:
Maybe we want to add a short dev comment here that this is a critical respectively sensitive error message where the text string is needed to remain stable for other core functionality to work.